modules/gapi/misc/python/test/test_gapi_sample_pipelines.py

```diff
@@ -517,8 +517,8 @@ def run(arr0, arr1):
 
             comp = cv.GComputation(cv.GIn(g_arr0, g_arr1), cv.GOut(g_out))
 
-            arr0 = [(2, 2), 2.0]
-            arr1 = [3,    'str']
+            arr0 = ((2, 2), 2.0)
+            arr1 = (3,    'str')
```

Comment:
This extra change is necessary after #20618 (list -> tuple)

(or change `self.assertEqual(arr0 + arr1, out)` below)

/cc @VadimLevin

Reply:
Maybe: `self.assertSequenceEqual(arr0 + arr1, out)` would be better. Example:
```python
        from itertools import chain
        a = [1, 2, [1, 2]]
        b = ['str', (2, 2)]
        res = tuple(chain(a, b))
        b[0] = 'a' # This modification is required to fail assertion below
        self.assertSequenceEqual(a + b, res)
```
reports

> AssertionError: Sequences differ: [1, 2, [1, 2], 'a', (2, 2)] != (1, 2, [1, 2], 'str', (2, 2))
> 
> First differing element 3:
> 'a'
> 'str'
> 
> - [1, 2, [1, 2], 'a', (2, 2)]
> + (1, 2, [1, 2], 'str', (2, 2))